test/shared/phpunit.xml

```diff
@@ -11,7 +11,6 @@
       <file>test/DMLTest.php</file>
       <file>test/MockTest.php</file>
       <file>test/MsgPackTest.php</file>
-      <file>test/AssertTest.php</file>
```

Comment:
I don't see any files removed in the PR. Looks like the file is still there.

Reply:
It was duplicate of the line: https://github.com/tarantool/tarantool-php/blob/3ebf76b20c28916e7f5bbe43da3bcb34dd7a42b5/test/shared/phpunit.xml#L9

Reply:
I've updated the commit message to make it clear.